ocrd_network/ocrd_network/processing_worker.py

```diff
@@ -51,6 +51,8 @@ def __init__(self, rabbitmq_addr, mongodb_addr, processor_name, ocrd_tool: dict,
         file_handler.setFormatter(logging.Formatter(logging_format))
         file_handler.setLevel(logging.DEBUG)
         self.log.addHandler(file_handler)
+        # TODO: remove this when refactoring the logging
+        self.log.setLevel(logging.DEBUG)
```

Comment:
I thought I had removed that in #1101 - need to check again

Reply:
The merge of master with #1101 to #1083 completely broke the logging in `ocrd_network`. Thus, I reverted the merge.

Reply:
This is now resolved with #1105.